### ISSUE TITLE
Attributes on new line not working

### ISSUE THREAD

**Issue by zerobytes** [DotJoshJohnson/vscode-xml#116]
Using VSCode 1.13.1
When xmlTools.splitXmlnsOnFormat is true, it only create new lines for the first not attributes.
All XML content will have its attributes remaining on the same line (necessarily).
Example

```
<View 
	xmlns:mvc="sap.ui.core.mvc" 
	xmlns="sap.m" 
	xmlns:core="sap.ui.core" 
	xmlns:tnt="sap.tnt" class="sapUiSizeCozy" controllerName="sap.cm.deal.capture.controller.InterStrategy">
	<Page id="idMainContent" showHeader="false">
		<content>
			<Panel height="100%">
				<headerToolbar>
					<Toolbar height="auto">
						<VBox width="40%">
							<HBox class="sapUiSmallMarginTop" renderType="Bare">
								<Icon src="sap-icon://document-text" size="24px" class="sapUiTinyMarginEnd" />
								<Title text="Inter Strategy" titleStyle="H2" />
							</HBox>
							<Title text="ISDL0129DL1209" titleStyle="H1" />
							<ObjectStatus title="{i18n>createdAt}" visible="{= ${createdAt} ? true : false}" class="sapUiSmallMarginBottom" text="{path:'createdAt' , type: 'sap.ui.model.type.Date', formatOptions: { style: 'long' }}, {createdBy}" />
						</VBox>
					</Toolbar>
				</headerToolbar>
				<content>
					<Title text="AASD"></Title>
				</content>
			</Panel>
		</content>
		<footer>
			<Toolbar>
				<Button id="idMessagePopOverButton" icon="sap-icon://message-popup" text="{messageManager>/length}" type="{parts: ['messageManager>/length', 'viewConfigs>/bMessagePopOverError'], formatter:'.oControls.oFormatter.formatPopOverMessage'}" press="onPopoverPressed" />
				<ToolbarSpacer />
				<Button type="Default" press="onCancelPressed" icon="sap-icon://sys-cancel" text="{i18n>cancel}" />
				<Button type="Default" icon="sap-icon://add-document" text="{i18n>saveAsDraft}" id="idSaveDraft" press="saveDraft" />
				<Button type="Default" icon="sap-icon://create-session" press="saveAsTemplate" id="idSaveAsTemplate" text="{i18n>saveAsTemplate}" />
				<Button id="idPublishButton" type="Accept" press="publishDeal" icon="sap-icon://upload-to-cloud" text="{i18n>previewAndPublish}" />
				<Button id="idAmendButton" type="Accept" press="amendDeal" icon="sap-icon://upload-to-cloud" text="{i18n>previewAndAmend}" visible="false" enabled="false" />
				<Button id="idUpdateButton" type="Accept" press="updateTemplate" icon="sap-icon://upload-to-cloud" text="{i18n>update}" visible="false" enabled="false" />
			</Toolbar>
		</footer>
	</Page>
</View>
```

**Comment by nknganda** [comment]
Would also love to have an option that allows having attributes on newlines. 
For example:

        <report 
            id="report_web_quote"
            string="Web Quotation"
            model="sale.order" 
            report_type="qweb-pdf"
            file="website_quote.report_quotation" 
            name="website_quote.report_quote" 
            menu="False"
        />

**Comment by wprater** [comment]
looking forward to seeing this as well!

**Comment by DotJoshJohnson** [comment]
Released in [`v2.0.0-preview.1`](https://github.com/DotJoshJohnson/vscode-xml/releases/tag/v2.0.0-preview.1). This will be released to the marketplace when `v2.0.0` is out of preview.
